work-docs/02-non-page-spec.md

```
@@ -136,12 +136,27 @@ docs/               確定したドキュメント（必要になったら）
-- 共通レイアウト（`apps/web/src/layouts/BaseLayout.astro` 等）の `</head>` または `</body>` 直前に公式スクリプトを挿入
+- Cloudflare Web Analytics は「Cloudflare Pages の one-click setup（自動挿入）」と「手動で script を埋め込む」の 2 方式がある
+  - 本プロジェクトは **ローカル/CI でも同じ HTML が生成される**ことを優先し、手動で script を埋め込む方式を採用する
+- 共通レイアウト（`apps/web/src/layouts/BaseLayout.astro`）の `</head>` 内に公式スクリプトを挿入する
 - `defer` を付け、描画のブロックを避ける
+- SSG（MPA）前提のため SPA Measurement は不要とし、`data-cf-beacon` で `spa: false` を明示する（History API のフックや `popstate` 監視を避ける）
 
 設定方針:
 - Web Analytics の token は秘匿情報ではないが、差し替えやすいように `PUBLIC_CF_ANALYTICS_TOKEN` として環境変数化する（Astro 側は `import.meta.env.PUBLIC_CF_ANALYTICS_TOKEN`）
 
+挿入するスクリプト（例）:
+```html
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)